pkgs/applications/window-managers/i3/pystatus.nix

```diff
@@ -1,14 +1,17 @@
-{ stdenv, fetchurl, libpulseaudio, python3Packages, extraLibs ? [] }:
+{ stdenv, fetchFromGitHub, libpulseaudio, python3Packages, extraLibs ? [] }:
 
 python3Packages.buildPythonApplication rec {
   name = "${pname}-${version}";
-  version = "3.35";
+  version = "2018-04-11-unstable";
```

Comment:
Please switch order of "unstable" and the date.
Also add a comment that they're using a rolling release model and link to https://github.com/enkore/i3pystatus/issues/584#issuecomment-318085048.